src/controllers/commands/builtin/chatterino/Debugging.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+class QString;
+
+namespace chatterino {
+
+struct CommandContext;
+
+namespace commands {
+
+    QString setLoggingRules(const CommandContext &ctx);
+
+}  // namespace commands
+
+}  // namespace chatterino
```

Comment:
I would rather the forward-declaration is done in a separate namespace definition - see ShieldMode.hpp